packages/inventory-compliance/package.json

```diff
@@ -44,9 +44,8 @@
         "@redhat-cloud-services/frontend-components-remediations": "*",
         "@redhat-cloud-services/frontend-components-utilities": "*",
         "@apollo/react-hooks": "^3.1.5",
-        "apollo-boost": "^0.1.23",
-        "graphql-tag": "^2.10.0",
-        "react-apollo": "^2.3.3",
```

Comment:
What's behind the `react-apollo` removal?

Reply:
We don't need it anymore `react-apollo` used to be the overarching Apollo package, but we have been moving to `@apollo/*` packages over time.